### PR TITLE
Use "git symbolic-ref" to determine active Git branch

### PR DESCRIPTION
**Pull request by mkenne11**
Modified the "branches" function to use the "git symbolic-ref --short -q HEAD" command to determine the active Git branch.

### CODE REVIEW

**Comment by drew-gross** [comment]
Awesome, thanks for the help!

**Comment by mkenne11** [comment]
No worries at all happy to help!

I'm not super on top of asynchronous calls in ts/js. See my question inline about the two async calls which "lines.map" is dependent on.

File: src/utils/Git.ts [hunk 1]
```diff
@@ -121,12 +122,20 @@ export async function branches({
...
+        ['"symbolic-ref"', '"--short"', '"-q"', '"HEAD"'],
+        directory)
+        .then(branchName => branchName.trim());
     let lines = await linedOutputOf(
         "git",
         ["for-each-ref", tags ? "refs/tags" : "", "refs/heads", remotes ? "refs/remotes" : "", "--format='%(HEAD)%(refname:short)'"],
         directory,
     );
-    return lines.map(line => new Branch(line.slice(1), line[0] === "*"));
+    return lines.map(line => {
```
Comment (mkenne11) [inline src/utils/Git.ts]: Can I be sure that both the "currentBranch" and "lines" asynchronous calls always finish before "lines.map" code runs?

Comment (drew-gross) [inline src/utils/Git.ts]: Yep. The `await` guarantees that.

Comment (mkenne11) [inline src/utils/Git.ts]: Thanks for the response.

**Pull request merged**